cmd/amqpproxy/main_test.go

```diff
@@ -140,6 +165,14 @@ func mustCreateAMQPProxy(t *testing.T, args []string) *testAMQPProxy {
 	return tfi
 }
 
+func mustCreateAMQPProxyExcludePayloadData(t *testing.T, args []string) *testAMQPProxy {
+	args = []string{"--exclude-payload-data"}
+	t.Logf("Creating AMQP Proxy with %#v", args)
+
+	tfi := mustCreateAMQPProxy(t, args)
```

Comment:
There's only a single test that uses this, so I'd just inline this in the test above.

So just convert the call to mustCreateAMQPProxyExcludePayloadData to mustCreateAMQPProxy(t, []string{"--exclude-payloadd-data"})